testdata/list_users.json

```diff
@@ -63,21 +71,29 @@
             "createdAt": "1234567890000",
             "lastLoginAt": "1233211232000",
             "customAttributes": "{\"admin\": true, \"package\": \"gold\"}",
-            "tenantId": "testTenant"
+            "tenantId": "testTenant",
+            "mfaInfo": [
+              {
+                "phoneInfo": "+31612345678",
```

Comment:
Nit: Suggest using `+1234567890` as the phone number and `testuser` as the enrollment ID to make it obvious that they are test data.